packages/org.json_schema.contrib/tests/ModuleGenerator.pkl-expected.pcf

```diff
@@ -0,0 +1,240 @@
+examples {
+  ["basic"] {
+    """
+    /// This module was generated from JSON Schema from <>.
```

Comment:
Should `baseUri` not actually be required `!isEmpty`?

Reply:
`baseUri` is a `pkl.experimental.uri.URI`, not a string. Practically, though, users will use the generator by passing the JSON Schema URI as a CLI flag, and the generator uses `read()` to get the schema. We can add validation here, but I don't know if it's actually useful.